wdio.conf.js

```diff
@@ -0,0 +1,45 @@
+/* global browser */
+/* eslint-disable import/no-extraneous-dependencies */
+const wdioConf = require('terra-toolkit/lib/wdio/conf');
+const WebpackDevService = require('terra-toolkit/lib/wdio/services/index').WebpackDevService;
+const localIP = require('ip');
+const path = require('path');
+const webpackConfig = require('./packages/terra-site/webpack.config.js');
+
+const webpackPort = 8080;
+
+// Flex specs search between local pacakge and repo
+let specs = path.join('tests', 'wdio', '**', '*-spec.js');
+if (__dirname === process.cwd()) {
+  specs = path.join('packages', '**', specs);
+}
+
+const config = {
+  ...wdioConf.config,
+
+  baseUrl: `http://${localIP.address()}:${webpackPort}`,
+  specs,
+
+  seleniumDocker: {
+    enabled: !process.env.TRAVIS,
+  },
+
+  webpackPort,
+  webpackConfig,
+  before() {
+    // eslint-disable-next-line prefer-arrow-callback, func-names
+    browser.addCommand('setCSSCustomProps', function (properties) {
```

Comment:
this needs to move to toolkit... but i'd like to push this through and do a PR this weekend to put it there if possible. I'd like to wrap up the badge PR unless anyone feels strongly about it

Reply:
I'm fine with moving this to toolkit in a follow-up PR.